buildSrc/build.gradle

```diff
@@ -16,7 +16,7 @@ dependencies {
     implementation 'io.spring.dependency-management:io.spring.dependency-management.gradle.plugin:1.1.4'
     implementation 'org.hidetake.swagger.generator:org.hidetake.swagger.generator.gradle.plugin:2.19.2'
     implementation 'org.sonarqube:org.sonarqube.gradle.plugin:4.4.1.3373'
-    implementation 'org.springframework.boot:spring-boot-gradle-plugin:2.7.18'
+    implementation 'org.springframework.boot:spring-boot-gradle-plugin:3.1.2'
```

Comment:
Reminder for the Workspaces team to try re-enabling dependabot updates for sprint-boot libraries: https://github.com/DataBiosphere/terra-landing-zone-service/blob/608d7b7d88daea05577baf3eb7ed428dbfc0574a/.github/dependabot.yml#L4.